docs/packaging/prepare-for-packaging.md

```diff
@@ -77,11 +77,11 @@ sudo solbuild update
 
 ## Setting up a Build Directory
 
-Create a packaging directory to hold sub-directories with packaging files. Here we are using the name `solus-builds` and placing it in our home directory, but you can use whatever you prefer.
+Create a packaging directory to hold sub-directories with packaging files. Here we are using the name `solus-packages` and placing it in our home directory, but you can use whatever you prefer.
 
 ```bash
-mkdir ~/solus-builds
-cd solus-builds
+mkdir ~/solus-packages
+cd solus-packages
 ```
```

Comment:
As said in the description, the clone command makes this section completely redundant. I would vote to remove this section and clarify in the cloning section that the directory will be created if it doesn't exist.